### PR TITLE
Removes plastic tiles and large wood tiles

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Removes plastic floor tiles & large wood tiles from the game.

Not tested ingame; I have no clue what testing this would even help with that tests wouldn't, though.

## Why / Balance
Mappers *constantly* accidentally place plastic tiles instead of real ones. You can find them all over the place on marathon, for instance. They're virtually indistinguishable unless you look closely, and have no real reason to exist because of that.

Large wood floor tiles don't fit the style of other tiles & were merged without art director approval, and we've been telling people not to use them in maps for quite a while. Though it is *maybe* possible they could be salvaged with a resprite, I sincerely doubt it would look good considering how large floor tiles look ingame already. The amount of negative space present on them is just not very good looking.

## Technical details
N/A

## Media
N/A

## Requirements
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

Probably not applicable???

**Changelog**
no cl no n

### CODE REVIEW

**Comment by chavonadelal** [comment]
«it's easier to break than to make»

File: Resources/Prototypes/Entities/Objects/Misc/tiles.yml [hunk 1]
```diff
@@ -1522,23 +1522,4 @@
     - Plating
     - FloorAstroSnow
   - type: Stack
-    stackType: FloorTileAstroSnow
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Misc/tiles.yml:1525]: Don't remove this line. It belongs to the previous entity.

**Comment by Emisse** [comment]
dont remove plastic tiles

**Comment by UbaserB** [comment]
> dont remove plastic tiles

can i ask why

**Comment by Emisse** [comment]
i use them to distinguish different areas for walking etc ie under firelocks, medical pathways through triage, etc. theyre subtle but they help subconsciously guide people

**Comment by Flareguy** [comment]
Will make another PR to address emisse's requested changes later, since it'd probably be easier to just start over.

**Pull request closed without merging**